### PR TITLE
Bump nuitka

### PR DESCRIPTION
**Pull request by oz123**
dev-python/nuitka: bump version to 0.6.6

Closes: https://bugs.gentoo.org/705392
Package-Manager: Portage-2.3.79, Repoman-2.3.16
Signed-off-by: Oz Tiram <oz.tiram@gmail.com>

### CODE REVIEW

File: dev-python/nuitka/nuitka-0.6.6.ebuild [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+HOMEPAGE="https://www.nuitka.net"
+SRC_URI="https://nuitka.net/releases/${P^}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="dev-python/appdirs[${PYTHON_USEDEP}]
+	>dev-util/scons-3.0.5[${PYTHON_USEDEP}]
+	dev-python/setuptools[${PYTHON_USEDEP}]"
```
Comment (juippis) [inline dev-python/nuitka/nuitka-0.6.6.ebuild]: Please use  `DISTUTILS_USE_SETUPTOOLS` with this one.

File: dev-python/nuitka/nuitka-0.6.6.ebuild [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+S="${WORKDIR}/${P^}"
+
+pkg_postinst() {
+	optfeature "support for stand-alone executables" app-admin/chrpath
+}
```
Comment (juippis) [inline dev-python/nuitka/nuitka-0.6.6.ebuild]: I see upstream changelog mentions: 
```
Standalone: Added support for dill.

Standalone: Added support for scikit-image and skimage.

Standalone: Added support for weasyprint.

Standalone: Added support for dask.

Standalone: Added support for pendulum.

Standalone: Added support for pytz and pytzdata.
```

Does this mean anything? More `optfeature` perhaps?

File: dev-python/nuitka/nuitka-0.6.6.ebuild [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+DESCRIPTION="Python to native compiler"
+HOMEPAGE="https://www.nuitka.net"
+SRC_URI="https://nuitka.net/releases/${P^}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="dev-python/appdirs[${PYTHON_USEDEP}]
+	>dev-util/scons-3.0.5[${PYTHON_USEDEP}]
```
Comment (juippis) [inline dev-python/nuitka/nuitka-0.6.6.ebuild]: Always satisfied, version requirement not needed.

File: dev-python/nuitka/nuitka-0.6.6.ebuild (hunk 2 above)
Comment (oz123) [reply dev-python/nuitka/nuitka-0.6.6.ebuild]: These are libraries which can be distributed with the application. They need explicit support in nuitka. 
According to the source code, [standalone still only needs chrpath.](https://github.com/Nuitka/Nuitka/blob/00748128bc6acecfcb92e0a4aeaeee0c34f266c7/nuitka/freezer/Standalone.py#L1364).

File: dev-python/nuitka/nuitka-0.6.6.ebuild (hunk 3 above)
Comment (oz123) [reply dev-python/nuitka/nuitka-0.6.6.ebuild]: done.

File: dev-python/nuitka/nuitka-0.6.6.ebuild (hunk 1 above)
Comment (oz123) [reply dev-python/nuitka/nuitka-0.6.6.ebuild]: OK, I removed this and added this line.
 DISTUTILS_USE_SETUPTOOLS=rdepend.

**Comment by juippis** [comment]
```
 * Messages for package dev-python/nuitka-0.6.6:

--- replaced obj /usr/lib64/python3.6/site-packages/nuitka/nodes/FrameNodes.py
 *   app-admin/chrpath for support for stand-alone executables
 *
 * This package installs one or more Python modules that are not byte-compiled.
 * The following files are missing:
 *
 *   /usr/lib64/python3.6/site-packages/nuitka/build/inline_copy/pefile/ordlookup/__pycache__/__init__.cpython-36.pyc
 *   /usr/lib64/python3.6/site-packages/nuitka/build/inline_copy/pefile/ordlookup/__pycache__/__init__.cpython-36.opt-1.pyc
 *   /usr/lib64/python3.6/site-packages/nuitka/build/inline_copy/pefile/ordlookup/__pycache__/__init__.cpython-36.opt-2.pyc
 *   /usr/lib64/python3.6/site-packages/nuitka/build/inline_copy/pefile/ordlookup/__pycache__/oleaut32.cpython-36.pyc
 *   /usr/lib64/python3.6/site-packages/nuitka/build/inline_copy/pefile/ordlookup/__pycache__/oleaut32.cpython-36.opt-1.pyc
 *   /usr/lib64/python3.6/site-packages/nuitka/build/inline_copy/pefile/ordlookup/__pycache__/oleaut32.cpython-36.opt-2.pyc
 *   /usr/lib64/python3.6/site-packages/nuitka/build/inline_copy/pefile/ordlookup/__pycache__/ws2_32.cpython-36.pyc
 *   /usr/lib64/python3.6/site-packages/nuitka/build/inline_copy/pefile/ordlookup/__pycache__/ws2_32.cpython-36.opt-1.pyc
 *   /usr/lib64/python3.6/site-packages/nuitka/build/inline_copy/pefile/ordlookup/__pycache__/ws2_32.cpython-36.opt-2.pyc
 *
 * Please either fix the upstream build system to byte-compile Python modules
 * correctly, or call python_optimize after installing them.  For more
 * information, see:
 * https://wiki.gentoo.org/wiki/Project:Python/Byte_compiling
 *
```
Could you look into this? The fix seems like an easy one.

**Comment by oz123** [comment]
@juippis my last commit added `python_optimize` so now all files should be compiled.

**Pull request closed without merging**